docs/onboarding.rst

```diff
@@ -142,11 +142,11 @@ Architecture Overview
     Raiden Architecture Overview
 
 
-At the core of the Raiden architecture lies a state machine. The state machine gets fed state changes from various sources such as:
+At the core of the Raiden architecture lies a state machine. The state machine gets fed state changes from various sources.
 
-- User commands directly from the user through the REST API
-- Trigerred via blockchain via polling for blockchain events
-- Triggered by receiving an offchain message
+- ``Action*`` state changes are caused by user commands directly from the user through the REST API.
+- ``ContractReceive*`` state changes are caused by blockchain events fetched via polling.
+- ``Receive*`` state changes are caused by receiving an offchain message.
```

Comment:
Very nice @pirapira.

This is the nomenclature of `StateChange`s as pointed out in [here](https://github.com/raiden-network/raiden/blob/a3e735a4df821e12eeeed3c211fab8fe40192941/raiden/transfer/architecture.py#L65).

Can you also add the naming nomenclature for `Event`s? It's [here](https://github.com/raiden-network/raiden/blob/a3e735a4df821e12eeeed3c211fab8fe40192941/raiden/transfer/architecture.py#L88)

Reply:
Will do on this PR because they are simillar enough.

Reply:
I added descriptions of ``Send*`` ``ContractSend*`` and ``Event*``.  Please have a look.

Reply:
Thank you